### PR TITLE
Adding dotnet-projectmodel-server back

### PR DESCRIPTION
**Pull request by livarcocc**
Adding dotnet-projectmodel-server back because ASP.NET needs it to unblock themselves from start migrating to msbuild. We need to remove this as soon as ASP.NET can migrate all their projects to csproj and the tooling is in place.

@eerhardt @Sridhar-MS @brthor @piotrpMSFT @anurse

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX x64 Release Build

**Comment by brthor** [comment]
You need to add the command back here:
https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/Program.cs#L48

**Comment by livarcocc** [comment]
@brthor Good catch. Thanks Bryan.

File: src/dotnet/Program.cs [hunk 1]
```diff
@@ -45,7 +45,8 @@ public class Program
             ["run3"] = Run3Command.Run,
             ["restore3"] = Restore3Command.Run,
             ["pack3"] = Pack3Command.Run,
-            ["migrate"] = MigrateCommand.Run
+            ["migrate"] = MigrateCommand.Run,
```
Comment (brthor) [inline src/dotnet/Program.cs]: need `using Microsoft.DotNet.ProjectModel.Server`

Comment (livarcocc) [inline src/dotnet/Program.cs]: I know. Added that there.

**Comment by anurse** [comment]
LGTM. Thanks!

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

**Comment by eerhardt** [comment]
:shipit:

**Comment by anurse** [comment]
:shipit:

**Comment by eerhardt** [comment]
test Windows_NT x64 Release Build
test Windows_NT x86 Debug Build

**Comment by eerhardt** [comment]
Livar is on vacation.  I'll merge this when I get green.

**Comment by anurse** [comment]
test Windows_NT x64 Release Build

**Comment by anurse** [comment]
Thanks @eerhardt !

**Pull request merged**